core/embed/rust/src/ui/model_tt/component/frame.rs

```diff
@@ -121,12 +121,14 @@ where
         const TITLE_SPACE: i16 = theme::BUTTON_SPACING;
 
         let bounds = bounds.inset(self.border);
+        // Allowing for little longer titles to fit in
+        let title_extra_space = Insets::right(2);
```

Comment:
Nit
```suggestion
        const TITLE_EXTRA_SPACE: Insets = Insets::right(2);
```

Reply:
Changed in the force-pushed commit [1cf2c52](https://github.com/trezor/trezor-firmware/pull/3145/commits/1cf2c52b4045e522201d8a88fb9c67d296074937)